statements/src/test/java/org/lukawska/trainsmart/statements/infra/config/StatementsDefinitionTest.java

```diff
@@ -0,0 +1,65 @@
+package org.lukawska.trainsmart.statements.infra.config;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.lukawska.trainsmart.statements.testutil.StatementTestData.*;
+
+@ExtendWith(MockitoExtension.class)
+class StatementsDefinitionTest {
+
+    private StatementsDefinition statementsDefinition;
+
+    @Test
+    void shouldReturnStatementWhenPresent() {
+        //given
+        final String statementCode = randomStatementCode();
+        statementsDefinition = new StatementsDefinition();
+        Statement statement = requiredStatement();
+        Map<String, Statement> defMap = Map.of(statementCode, statement);
+        statementsDefinition.setStatements(defMap);
+
+        //when
+        Optional<Statement> found = statementsDefinition.findStatementByCode(statementCode);
+
+        //then
+        assertThat(found).isPresent()
```

Comment:
bez sensu dzielic na linijki takie rzeczy. jak cos sie miesci w jednej linii i nie przekracza 120 znakow to zostaw to w jednej linijce. Wyjatek to streamy i buildery